codalab/server/bundle_manager.py

```diff
@@ -363,17 +374,6 @@ def _filter_and_sort_workers(workers_list, bundle, bundle_resources):
             worker for worker in workers_list if worker['memory_bytes'] >= bundle_resources.memory
         ]
 
-        # Filter by tag.
-        request_queue = bundle.metadata.request_queue
-        if request_queue:
-            tagm = re.match('tag=(.+)', request_queue)
-            if tagm:
-                workers_list = [worker for worker in workers_list if worker['tag'] == tagm.group(1)]
-            else:
-                # We don't know how to handle this type of request queue
-                # argument.
-                return []
```

Comment:
We should add a logging statement here about how we got a malformed `request_queue`